Arm/Cosim.lean

```diff
@@ -267,14 +267,10 @@ def run_n_tests (verbose : Bool) (logPrefix : String)
 
 /-- Run n random tests for each random instruction generator in
 `Insts.rand`. -/
-def run_all_tests (verbose : Bool) (n : Nat) : IO UInt32 := do
+def run_all_tests_go (verbose : Bool) (n : Nat) : Cosim.CosimM UInt32 := do
   IO.println s!"Performing conformance testing..."
-  let machine_check ←
-    IO.Process.output
-    { cmd  := "Arm/Insts/Cosim/platform_check.sh",
-      args := #["-m"] }
   let mut tasks := #[]
-  if machine_check.exitCode == 1 then
+  if ← Cosim.aarch64? then
```

Comment:
Non mechanical change, please review

Reply:
Sometimes you put parentheses around `← Cosim.aarch64?`. It probably doesn't matter, just an observation for the inconsistency.

Reply:
Fixed, I've opted to bing Lean-like, so no parens. Sorry Shilpi! (Don't worry, I love Lisp too 😄 )